Navigation-Examples/Examples/Building-Extrusion.swift

```diff
@@ -0,0 +1,195 @@
+import UIKit
+import MapboxCoreNavigation
+import MapboxNavigation
+import MapboxDirections
+import Mapbox
+
+class BuildingExtrusionViewController: UIViewController, NavigationMapViewDelegate, NavigationViewControllerDelegate, MGLMapViewDelegate {
+    
+    var mapView: NavigationMapView?
+    
+    var routeOptions: NavigationRouteOptions?
+    
+    var currentRoute: Route? {
+        get {
+            return routes?.first
+        }
+        set {
+            guard let selected = newValue else { routes?.remove(at: 0); return }
+            guard let routes = routes else { self.routes = [selected]; return }
+            self.routes = [selected] + routes.filter { $0 != selected }
+        }
+    }
+    
+    var routes: [Route]? {
+        didSet {
+            guard let routes = routes, let current = routes.first else { mapView?.removeRoutes(); return }
```

Comment:
I think it's better to call `removeRoutes()` instead
```suggestion
            guard let routes = routes, let current = routes.first else { removeRoutes(); return }
```

Reply:
Yes, I think it was handled in most recent update.